### DISCUSSION TITLE
Old PC profile didnt copy settings

### DISCUSSION THREAD

**Discussion by marijnx053** [runelite/runelite#17097] [Q&A]
Did my yearly windows reset, installed runelite, loggin in.
None of my plugins/settings were taken to the new pc.

Weird, must have done something wrong. 
But then I noticed it did take my loot tracker.

Any tips on how to get all my settings back?

**Comment by Felanbird** [comment]
https://github.com/runelite/runelite/wiki/FAQ#my-plugins-are-gone-or-my-config-profile-is-missing

**Reply by marijnx053** [reply]
Thank you!! I will try that
